src/editor/components/StylesColor.js

```diff
@@ -51,7 +97,14 @@ const Color = ( { selector } ) => {
 			<span className="themer--styles__item__title">
 				{ __( 'Color', 'themer' ) }
 			</span>
-			<div className="themer--styles__item__columns themer--styles__item__columns--2">
+			<ContrastChecker
+				textColor={ varToHex( textColour, themePalette ) }
+				backgroundColor={ varToHex( backgroundColour, themePalette ) }
```

Comment:
Debouncing these values is likely to be a more concise and stable solution to the jumping UI issue we were experiencing. Debouncing would cause the value to only update `x` milliseconds after it stops changing, and I would recommend this over the additional event handlers.

We should be able to use the debounce method from lodash (bundled with WP) to achieve this - https://lodash.com/docs/4.17.15#debounce

Reply:
I've updated the approach to use `debounce` and a demo of this in action can be [seen here](http://bigbite.im/v/Nzi8jq). It does add a minimal visual delay to the feedback, but it does remove a lot of complexity from the component that was added with the event listeners.

Thanks very much for the information!